runtime/src/bank.rs

```diff
@@ -1110,6 +1115,45 @@ impl Bank {
         )
     }
 
+    fn collect_rent(
```

Comment:
can we make this a method on the batch, instead?

Reply:
hmm. By batch you mean `Vec<TransactionLoadResult>` correct?
Three reasons, I chose to apply this on bank: 
1) `collect_fee` is also defined on bank, so consistency. 
2) If we do define on `Vec<TransactionLoadResult>`, we need to add safeguards there to make sure, that this can only be executed once (which adds extra complexity).